### DISCUSSION TITLE
" ? " in the file names can't be downloaded. Says "Remote File Error"

### DISCUSSION THREAD

**Discussion by GoldenGlimpse** [nicotine-plus/nicotine-plus#2243] [Q&A] [answered]
Hi

Is there a way to allow downloading files with a question mark in the title name?
When i try, it says "Remote File Error".
Or is the only option to ask the user to change the filename?

Nicotine+ 3.2.6, Linux.

thank you

**Comment by mathiascode** [comment, marked as answer]
We already do as much as we can on our end to fix broken downloads from legacy Soulseek clients. The only option left is to ask the user to change the filename or use a different client.

I just looked up some MP3 files that have a question mark in the filename. These files were all shared by Soulseek NS users, and didn't display any bitrate or duration, indicating that Soulseek NS never managed to open the files to read the metadata.
